single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -1226,17 +1216,22 @@ private void drawNodesInfos(Element root, VoltageLevelGraph graph,
         double yPos = graph.getY() - layoutParameters.getVoltageLevelPadding().getTop() + graph.getHeight() + CIRCLE_RADIUS_NODE_INFOS_SIZE;
 
         double xShift = graph.getX() + xInitPos;
-        for (ElectricalNodeInfo node : initProvider.getElectricalNodesInfos(graph)) {
-            String idNode = metadata.getSvgParameters().getPrefixId() + "NODE_" + node.getBusId();
+        for (NodeInfo nodeInfo : initProvider.getNodesInfos(graph)) {
```

Comment:
Maybe we could take advantage of this PR to change the name of the `initProvider` variable into `labelProvider`. Is there a reason to call it `initProvider ` @flo-dup ? Is it just legacy?

Reply:
it's just legacy indeed, so you could rename it

Reply:
fixed